programs/sbf/Cargo.toml

```diff
@@ -24,6 +24,7 @@ num-traits = "0.2"
 rand = "0.8"
 rustversion = "1.0.14"
 serde = "1.0.112"
+serde_derive = "1.0.112"
```

Comment:
Same here, can you add a comment to always keep this in sync with the "serde" version along with a link to the issue?